### PR TITLE
ui. add backup destination check

### PR DESCRIPTION
**Pull request by federicoballarini**
Refer to https://github.com/NethServer/dev/issues/5955

### CODE REVIEW

File: ui/src/components/system/Backup.vue [hunk 1]
```diff
@@ -3183,6 +3217,150 @@ export default {
...
+		    HostToPing: b.props.SftpHost
+		  }
+		break;
+		  
+		case "webdav":
+		  configObj = {
+		    WebDAVLogin: b.props.WebDAVLogin,
+		    WebDAVPassword: b.props.WebDAVPassword,
+		    WebDAVUrl: b.props.WebDAVUrl,
+		    HostToPing: 'nethserver.org' /* checking internet connection */
```
Comment (gsanchietti) [inline ui/src/components/system/Backup.vue]: Why are you checking internet connection?
The webdav could be local and the internet check is not related at all to the webdav access.

File: ui/src/components/system/Backup.vue [hunk 2]
```diff
@@ -3183,6 +3217,150 @@ export default {
...
+		  }
+		break;
+		  
+		case "b2":
+		  configObj = {
+            B2AccountId: b.props.B2AccountId,
+		    B2AccountKey: b.props.B2AccountKey,
+		    B2Bucket: b.props.B2Bucket,
+		    HostToPing: 'nethserver.org' /* checking internet connection */
```
Comment (gsanchietti) [inline ui/src/components/system/Backup.vue]: Why nethserver.org should be related to Backblaze server?
If nethserver.org is down, why Backblaze should be considered down too?

File: ui/src/components/system/Backup.vue [hunk 3]
```diff
@@ -3183,6 +3217,150 @@ export default {
...
+		  }
+	    break;
+		  	
+		case "s3":
+          configObj = {
+		    S3AccessKey: b.props.S3AccessKey,
+		    S3Bucket: b.props.S3Bucket,
+		    S3SecretKey: b.props.S3SecretKey,
+		    S3Host: b.props.S3Host,
+		    HostToPing: 'nethserver.org' /* checking internet connection */
```
Comment (gsanchietti) [inline ui/src/components/system/Backup.vue]: See previous comment.

File: ui/src/components/system/Backup.vue [hunk 4]
```diff
@@ -3183,6 +3217,150 @@ export default {
...
+        configObj,
+        null,
+        function(success) {
+          context.currentDataBackup.checkDestinationDescription = context.$i18n.t('backup.destination_is_reachable');
+          context.currentDataBackup.checkDestinationResult = 'success';
+        },
+        function(error) {
+          console.error(error);
+
+          if(configObj.HostToPing != null){
```
Comment (gsanchietti) [inline ui/src/components/system/Backup.vue]: If I understand correctly. this line means that no check is done for local resources like USB?

**Comment by gsanchietti** [comment]
If we want to implement such feature, please ensure also the commit history of the PR is clean:
- no duplicate messages are allowed
- merge commit should be avoided (use git rebase)
- respect commit message code conventions

**Comment by federicoballarini** [comment]
Close this PR and open a new clean one.

**Comment by gsanchietti** [comment]
Before opening this PR, please discuss the feature on community ;)

**Comment by federicoballarini** [comment]
Sure :)

**Pull request closed without merging**